packages/wallets/src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import moment from 'moment';
+import Calendar, { CalendarProps } from 'react-calendar';
+import CalendarIcon from '../../public/images/ic-calendar.svg';
+import FlowTextField, { TFlowFieldProps } from '../FlowField/FlowTextField';
+import unixToDateString, { customFormatShortWeekday } from './utils';
+import 'react-calendar/dist/Calendar.css';
+import './DatePicker.scss';
+
+interface TDatePickerProps extends TFlowFieldProps {
+    mobileAlignment?: 'above' | 'below';
+    onDateChange: (formattedDate: string | null) => void;
+    variant?: 'dateOfBirth' | 'expiry';
```

Comment:
Shouldnt we make the component more generic. I dont think we need these variants

Reply:
Updated 👍